google-beta/resource_compute_instance_group_manager.go

```diff
@@ -212,28 +193,28 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 						"max_surge_fixed": &schema.Schema{
 							Type:          schema.TypeInt,
 							Optional:      true,
-							Default:       1,
-							ConflictsWith: []string{"rolling_update_policy.0.max_surge_percent"},
+							Computed:      true,
```

Comment:
Because the Default was nonsensical to begin with. If `max_surge_percent` was set, `max_surge_fixed` would default to 1, which was an invalid value. `max_surge_percent` should only default to 1 if both `max_surge_percent` _and_ `max_surge_fixed` were not set, and the API would supply that default, anyways. Instead, `max_surge_fixed` defaulted to 1 if `max_surge_fixed` wasn't set, which isn't the same thing.

Reply:
makes sense, thanks!